admin-dev/themes/new-theme/scss/pages/_translation_page.scss

```diff
@@ -1,5 +1,162 @@
+$color-error: #c05c67 !default;
+$color-active: #25b9d7 !default;
+
 .export-translations {
   .export-translations-child {
     padding-left: 50px;
   }
 }
+
+.breadcrumb {
+  small {
+    color: $gray-medium;
+  }
+}
```

Comment:
Is the _translation_pages.scss on is own bundle ? Overwise this is wrong, it will override every breadcrumb on every pages, if it's intended, please move it to a global file, if not, then please use a parent selector to do it only on this page